src/Components/Navbar.tsx

```diff
@@ -0,0 +1,221 @@
+                        </g>
+                        <defs>
+                        <clipPath id="clip0_178_28">
+                        <rect width="60" height="60" fill="white" transform="translate(0.582397 0.498322)"/>
+                        </clipPath>
+                        </defs>
+                    </svg>
+                    <svg className="w-[15%]" viewBox="0 0 61 60" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fillRule="evenodd" clipRule="evenodd" d="M30.5897 60C47.1583 60 60.5897 46.5686 60.5897 30C60.5897 13.4315 47.1583 0 30.5897 0C14.0212 0 0.589722 13.4315 0.589722 30C0.589722 46.5686 14.0212 60 30.5897 60ZM45.2878 14.6991H15.2878C13.2253 14.6991 11.5566 16.3866 11.5566 18.4491L11.5378 40.9491C11.5378 43.0116 13.2253 44.6991 15.2878 44.6991H45.2878C47.3502 44.6991 49.0377 43.0116 49.0377 40.9491V18.4491C49.0377 16.3866 47.3502 14.6991 45.2878 14.6991ZM45.2878 22.1991L30.2878 31.5741L15.2878 22.1991V18.4491L30.2878 27.8241L45.2878 18.4491V22.1991Z" fill="white"/>
+                    </svg>
+                    <svg className="w-[15%]" viewBox="0 0 61 60" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fillRule="evenodd" clipRule="evenodd" d="M30.5969 60C47.1655 60 60.5969 46.5686 60.5969 30C60.5969 13.4315 47.1655 0 30.5969 0C14.0284 0 0.596924 13.4315 0.596924 30C0.596924 46.5686 14.0284 60 30.5969 60ZM30.2936 10.9491C23.0374 10.9491 17.1686 16.8179 17.1686 24.0741C17.1686 33.9179 30.2936 48.4491 30.2936 48.4491C30.2936 48.4491 43.4186 33.9179 43.4186 24.0741C43.4186 16.8179 37.5499 10.9491 30.2936 10.9491ZM30.2936 28.7616C27.7061 28.7616 25.6061 26.6616 25.6061 24.0741C25.6061 21.4866 27.7061 19.3866 30.2936 19.3866C32.8811 19.3866 34.9811 21.4866 34.9811 24.0741C34.9811 26.6616 32.8811 28.7616 30.2936 28.7616Z" fill="white"/>
+                    </svg>
+
+                    </div>
```

Comment:
Maybe use `w-8 h-8` or `w-12 h-12` instead of `w-[15%]` for icons
And change `<div className="absolute bottom-5 flex justify-around w-full p-5">` to `<div className="absolute bottom-5 flex justify-center space-x-5 px-auto">`

Before:
![image](https://github.com/ITeapotPL/iteapot-page/assets/26386606/393554ad-c3aa-401c-a847-e5d83f882d2c)

After:
![image](https://github.com/ITeapotPL/iteapot-page/assets/26386606/3b7bfcb1-c059-4dd5-90ef-a419733cf7e1)
![image](https://github.com/ITeapotPL/iteapot-page/assets/26386606/84b83cc6-45e7-4edf-8f7c-3699c4c7235f)

Reply:
I think w-8 is too small for smartphones and user may has a problem to click them correctly, in my opinion we can stay with w-12